spec/stringProcessing/stripHTMLTagsSpec.js

```diff
@@ -0,0 +1,16 @@
+var stripHTMLTags = require( "../../js/stringProcessing/stripHTMLTags.js" );
+
+describe( "strips the HTMLtags from a string", function(){
+	it( "returns a string without HTMLtags", function(){
```

Comment:
I wonder what it does with a string with multiple consecutive spaces 
(like: "this &nbsp;&nbsp; &lt;b&gt; is &nbsp;&nbsp; a &lt;/b&gt; teststring")
please add tests for this.